Source/cryptography/tests/cryptography_test/CMakeLists.txt

```diff
@@ -35,12 +35,17 @@ include_directories(${CMAKE_CURRENT_LIST_DIR}/../../../cryptography)
 include_directories(${CMAKE_CURRENT_LIST_DIR}/../../../cryptography/implementation)
 include_directories($<INSTALL_INTERFACE:include/${NAMESPACE}>)
 
+set_target_properties(cgimptests PROPERTIES
+        CXX_STANDARD 11
+        CXX_STANDARD_REQUIRED YES
+    )
+
 target_link_libraries(cgimptests
         PRIVATE
         ${NAMESPACE}Cryptography
+        ${NAMESPACE}Plugins::${NAMESPACE}Plugins
         ssl
         crypto
-        -Wl,--warn-unresolved-symbols
```

Comment:
Since it was warning, I have not noticed this and while running test app got segfault. Actually this was due to missing of EXTERANL attribute settings for crypto implementation headers. Hence now changed this to default (show it as error, instead warning) to solve it compile time itself.

Reply:
Oke so removing this will make warnings - > errors... Which is good and preferred!